lib/bundle/cask_dumper.rb

```diff
@@ -16,13 +16,24 @@ def casks
             .uniq
     end
 
-    def dump(casks_required_by_formulae)
+    def dump(casks_required_by_formulae, describe: false)
+      to_line = lambda { |cask|
+        comment = desc_comment(cask) if describe
+        "#{comment}cask \"#{cask}\""
+      }
```

Comment:
I see the `describe` usage now. Should be fine to have `desc_comment` take a `describe` argument and just use `.map {|c| desc_comment(c, describe)`. I think the lambda usage as is harms rather than helps readability.